src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngineUnitTest.java

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.samtools.TextCigarCodec;
+import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.vcf.VCFConstants;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.FeatureInput;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.walkers.annotator.interfaces.AnnotationGroup;
+import org.broadinstitute.hellbender.utils.genotyper.PerReadAlleleLikelihoodMap;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.*;
+
+public final class VariantAnnotatorEngineUnitTest extends BaseTest {
+    @Test
+    public void testEmpty(){
+        final List<String> annotationGroupsToUse= Collections.emptyList();
+        final List<String> annotationsToUse= Collections.emptyList();
+        final List<String> annotationsToExclude= Collections.emptyList();
+        final FeatureInput<VariantContext> dbSNPBinding = null;
+        final VariantAnnotatorEngine vae = VariantAnnotatorEngine.ofSelected(annotationGroupsToUse, annotationsToUse, annotationsToExclude, dbSNPBinding);
+        Assert.assertTrue(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().isEmpty());
+    }
+
+    @Test
+    public void testExclude(){
+        final List<String> annotationGroupsToUse= Collections.emptyList();
+        final List<String> annotationsToUse = Arrays.asList(Coverage.class.getSimpleName());//good one
+        final List<String> annotationsToExclude= annotationsToUse;
+        final FeatureInput<VariantContext> dbSNPBinding = null;
+        final VariantAnnotatorEngine vae = VariantAnnotatorEngine.ofSelected(annotationGroupsToUse, annotationsToUse, annotationsToExclude, dbSNPBinding);
+        Assert.assertTrue(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().isEmpty());
+    }
```

Comment:
Add a test case for exclusion of an annotation with non-empty `annotationGroupsToUse`, and make sure the individualized exclusion has priority.